src/components/NewActionForm.js

```diff
@@ -39,7 +39,11 @@ export default class NewEventForm extends Component {
                 mode: "cors",
                 headers: { "Content-Type": "application/json" },
                 body: JSON.stringify(formData)
-              }).then(this.setState({ title: "", description: "" }));
+              })
+              .then(res => res.json())
+              .then(response => console.log(response))
+              .then(this.setState({ title: "", description: "" }))
```

Comment:
SO we're echoing out the response but not using it, right?

Reply:
And then we empty the title and description out so the form is empty again?  I think that makes sense

Reply:
Yea were just checking if the server responds with 201 here